glnx-errors.c

```diff
@@ -23,7 +23,23 @@
 #include <glnx-backport-autocleanups.h>
 #include <glnx-errors.h>
 
-void
+gboolean
+glnx_throw (GError    **error,
+            const char *fmt,
+            ...)
+{
+  if (error == NULL)
+    return FALSE;
+
+  va_list args;
+  va_start (args, fmt);
+  GError *new = g_error_new_valist (G_IO_ERROR, G_IO_ERROR_FAILED, fmt, args);
+  va_end (args);
+  g_propagate_error (error, g_steal_pointer (&new));
+  return FALSE;
+}
+
+static void
```

Comment:
I guess one could just as well inline the `_real_` functions into the wrappers. I'm assuming no-one is calling them directly.

Reply:
Unfortunately we do in one place in libostree...which we could fix, but anyways this LGTM, please push!

Reply:
Here's a new version of this patch which restores the `glnx_real_` prototypes in `glnx-errors.h` (before I'd made them `static` in the `.c` file) -- there seems no harm in keeping them around.